checker/specification/to_implement.md

```diff
@@ -770,6 +770,47 @@ a satisfies 0; b satisfies string;
 
 - Expected string, found 1
 
+#### Array map
```

Comment:
Unfortunately it seems I broke the event recording inside `.map` and other iteration based things. Will come back to at some point